map/framework.cpp

```diff
@@ -384,7 +384,10 @@ Framework::Framework(FrameworkParams const & params)
   , m_bmManager(*this)
   , m_isRenderingEnabled(true)
   , m_routingManager(RoutingManager::Callbacks([this]() -> Index & { return m_model.GetIndex(); },
-                                               std::bind(&Framework::GetCountryInfoGetter, this)),
+                                               std::bind(&Framework::GetCountryInfoGetter, this),
```

Comment:
Please, don't mix bind with lambdas. Could you please replace this bind by a short lambda, like the one above, for index?